invenio_communities/templates/semantic-ui/invenio_communities/details/header.html

```diff
@@ -40,7 +40,12 @@ <h2 class="ui header">{{ community.metadata.title }}</h2>
               <span class="label label-keyword">{{ organization.name }}</span>
             {%- endfor %}
           {% endif %}
-
+      </div>
+      <div id="community-upload-button" class="four wide right floated right aligned community-header-actions column">
```

Comment:
is the `community-header-actions` class still needed?

Reply:
nope, thanks!